Sources/Commands/SwiftTestTool.swift

```diff
@@ -461,70 +461,6 @@ public struct SwiftTestTool: SwiftCommand {
         }
     }
 
-    /// Locates XCTestHelper tool inside the libexec directory and bin directory.
```

Comment:
This code is not removed but is moved out into a separate file where it can be used by both SwiftTestTool and SwiftPackageTool.